frontend/src/container/GuestUpload.jsx

```diff
@@ -181,9 +185,11 @@ class GuestUpload extends React.Component {
     if (!this.validateForm()) return false
 
     const formData = new FormData()
-
-    state.fileToUploadList.forEach((uploadFile, index) => {
-      formData.append(`file_${index}`, uploadFile)
+    // TODO - SG - 2021-01-22 - We really should use the uploadFile() function here
```

Comment:
Please create an issue about this and copy the link in the comment

Reply:
Issue created (https://github.com/tracim/tracim/issues/4090), I'll add the comment in #4088.